cmd/version/version.go

```diff
@@ -34,8 +32,8 @@ var (
 	buildDate = "1970-01-01T00:00:00Z" // build date in ISO8601 format, output of $(date -u +'%Y-%m-%dT%H:%M:%SZ')
 )
 
-// Version contains all the information related to the CLI version
-type Version struct {
+// version contains all the information related to the CLI version
+type version struct {
 	KubeBuilderVersion string `json:"kubeBuilderVersion"`
```

Comment:
```suggestion
	ToolVersion string `json:"toolVersion"`
```

Reply:
Just a nit since it can be used by another cli as well.

Reply:
No, that is only for kubebuilder, and it needs to stay as is because it is filled by `ldflags`